sdk/python/feast/feature_store_config.py

```diff
@@ -23,9 +23,9 @@ class Config:
 
     def __init__(
         self,
-        provider: Optional[str],
-        online_store: Optional[str],
-        metadata_store: Optional[str],
+        provider: Optional[str] = None,
+        online_store: Optional[str] = None,
+        metadata_store: Optional[str] = None,
     ):
         self.provider = provider if (provider is not None) else "local"
         self.online_store = online_store if (online_store is not None) else "local"
```

Comment:
I think we need a default `online_store` for the local provider, not a default online store regardless of provider.